### PR TITLE
Add more winter-appropriate garb to generic classes

### PR DESCRIPTION
**Pull request by pisskop**
Part 1?

Most bionic and RP-specific classes I left alone, but for generic classes I added pants, a light jacket, possibly gloves, and in some cases a light snack.  Some professions that seem to imply an easy to carry screwdriver kit got that, and the tailor got some scrap cloth.

### CODE REVIEW

**Comment by Vidiveni** [comment]
Electrician has jumpsuit and t-shirt which both cover normal layer. 

While I like the variety, these are buffs overall, which I don't really prefer unless it's for professions that was considered underpowered for their point cost. I understand they don't need to be perfectly balanced, but I want many weak professions to choose between.

File: data/json/professions.json [hunk 1]
```diff
@@ -68,7 +71,8 @@
                 "boots_winter",
                 "knit_scarf",
                 "pockknife",
-                "water_clean",
+                "energy_drink",
+                "chips3",
```
Comment (nikheizen) [inline data/json/professions.json]: I don't understand the purpose of changing the standard survivor from having a bottle of water to an energy drink and chips. A regular water bottle fits the theme (nothing notable) much better.

**Comment by pisskop** [comment]
Now that the early game is getting harder, the addition of a cotton jacket to various professions allows them to not freeze to death as they make their walk from house to house or when they start in more action oriented starts.

Various other buffs are more flavor-oriented.  Austerity has its charm, and Id love to see a way to add multiple versions of professions.

**Comment by nikheizen** [comment]
A lot of these changes don't make any sense wrt. winter survival, or indeed the theme of the profession itself.
Baseball cap and medical gloves on sheltered survivor?
Scalpel makes sense for a doctor, but for a lab technician?
_Adrenaline injector_ for a medical resident?
If you want to buff the classes generally, you should probably change the name of the PR, most of these aren't winter specific.

**Comment by pisskop** [comment]
Scalpel makes no sense on a general practicioner.  Scalpel, or some small utility pairing knife or box cutter, however, does make sense on characters who have to deal with measuring minute amounts or handle packaged goods.

Baseball caps are hinderances to other helmets, and completely reasonable on a prepared, sheltered individual.  And Im not sure youd find a prepper who didnt have some sort of disposable gloves.

Adrenaline injector provides more flexibility to the class, and has no benefit to winter survival, youve got me there.

Wearing caps are a completely reasonable response to being evacuated.  Having small tools that are commonplace at the workplace is just as reasonable.  To split up a small PR or rename it because I updated some outdated occupations and suggested some changes to the basic survivor build is silly.

File: data/json/professions.json (hunk 1 above)
Comment (Coolthulhu) [inline data/json/professions.json]: Yeah, energy drink + chips evokes image of a dorito + dr. peppers drinking slob.
Rest of the equipment is generally more survivor stuff - not everyone carries a pocket knife and winter gear (not even in winter).

File: data/json/professions.json [hunk 2]
```diff
@@ -354,10 +371,12 @@
             "both": [
                 "tank_top",
                 "jeans",
+                "jumpsuit",
```
Comment (Coolthulhu) [inline data/json/professions.json]: Jumpsuit conflicts with jeans.

File: data/json/professions.json [hunk 3]
```diff
@@ -343,7 +360,7 @@
         "ident": "mechanic",
         "name": "Home Mechanic",
         "description": "Although you never got your driver's license, you've always loved cars.  At least now you'll never be wanting for materials.",
-        "points": 1,
+        "points": 2,
```
Comment (Coolthulhu) [inline data/json/professions.json]: No need

File: data/json/professions.json [hunk 4]
```diff
@@ -556,7 +577,7 @@
                 "winter_jacket_army",
                 "helmet_liner",
                 "helmet_army",
-                "balclava",
+                "mask_dust",
```
Comment (Coolthulhu) [inline data/json/professions.json]: What for?

File: data/json/professions.json [hunk 5]
```diff
@@ -936,14 +963,18 @@
         "points": -1,
         "items": {
             "both": [
-                "pants",
-                "knit_scarf",
-                "whiskey",
+                "jeans",
+                "socks_bag",
+                "footrags",
+                "hoodie",
```
Comment (Coolthulhu) [inline data/json/professions.json]: Hoodie is good stuff, shouldn't be handed out like that to -1 classes.

File: data/json/professions.json [hunk 6]
```diff
@@ -1521,8 +1556,10 @@
         "items": {
             "both": [
                 "jumpsuit",
+                "tshirt",
```
Comment (Coolthulhu) [inline data/json/professions.json]: Conflicts with jumpsuit

File: data/json/professions.json [hunk 7]
```diff
@@ -4298,7 +4341,6 @@
                 "jeans",
                 "socks",
                 "sneakers",
-                "gloves_fingerless",
```
Comment (Coolthulhu) [inline data/json/professions.json]: Why?

**Comment by Coolthulhu** [comment]
Also, rename the PR when you're making significant changes unrelated to the original topic.

**Comment by pisskop** [comment]
You guys are of course right.  Im trying to add too much, and too quickly.  Ill reconsider.

**Comment by Cyrano7** [comment]
I know this is closed but I have a side note to add. @pisskop If your trying to fine tune the starting professions it's best to try to do it by making them more realistic without making them to much easier. The beginning of the game should be a bit of a mad scramble. Every profession should not start with ideal gear.

**Pull request closed without merging**